### ISSUE TITLE
missing quotes

### ISSUE THREAD

**Issue by pabra** [dillonkearns/elm-ts-json#10]
If you have a custom type and Codec like this:
```elm
type MyType
    = MyType Int


codec : Codec MyType
codec =
    Codec.custom Nothing
        (\vMyType value ->
            case value of
                MyType int ->
                    vMyType int
        )
        |> Codec.namedVariant1 "MyType" MyType ( "needs-quotes", Codec.int )
        |> Codec.buildCustom
```
it will generate this TypeScript type (missing quotes around `missing-quotes`):
```typescript
{ needs-quotes : number; tag : "MyType" }
```
here is [an ellie](https://ellie-app.com/hYn7Gj2JYgha1)

BTW: thanks for this project :heart_eyes:

**Comment by dillonkearns** [comment]
This is a great catch, thank you for reporting the issue!

I don't have bandwidth to work on this at the moment (I'm working on getting the elm-pages v3 release ready right now). I'd welcome a pull request on this if you have time and interest, but no pressure!

**Comment by dillonkearns** [comment]
I released this in package version 2.1.1: https://github.com/dillonkearns/elm-ts-json/blob/main/CHANGELOG.md#211---2022-08-24

Thanks again!
